src/translations/maps/utils/visuallyIndexed.js

```diff
@@ -0,0 +1,28 @@
+import { arrayFilter } from '../../../helpers/array';
+
+/**
+ * Insert new indexes to the list.
+ *
+ * @private
+ * @param {Array} indexesList List of indexes.
+ * @param {Number} insertionIndex Position inside the actual list.
+ * @param {Array} insertedIndexes List of inserted indexes.
+ * @returns List with new mappings.
+ */
+export function getListWithInsertedItems(indexesList, insertionIndex, insertedIndexes) {
```

Comment:
I find this confusing because it is inconsistent with the names used in `BaseMap`. How about changing this to:

```
/**
 * Insert new values to the list.
 *
 * @private
 * @param {Array} list BaseMap's internal data structure
 * @param {Number} insertionIndex Position inside the list.
 * @param {Array} insertedValues Array of inserted values.
 * @returns List with new values.
 */
export function getListWithInsertedValues(list, insertionIndex, insertedValues) {
```

Reply:
For discussion.